shibata added the new strings in the Japanese version to the English version. (#83)
fujitake/vantiq-related: vantiq-cloud-infra-operations/terraform_aws/readme_en.md

- locals  
  - `region`: Region to create  

>   - `worker_access_ssh_key_name`: Specify the name of the SSH key created in the Preparation (for worker node access).

Reviewer: ```suggestion
  - `worker_access_ssh_key_name`: Specify the name of the SSH key created in the Preparation (for accessing the worker node).
```

>   - `basion_access_ssh_key_name`: Specify the name of the SSH key created in the Preparation (for Bastion server access).

Reviewer: ```suggestion
  - `basion_access_ssh_key_name`: Specify the name of the SSH key created in the Preparation (for accessing the Bastion host).
```

Reviewer: bastion serverはすべてbastion hostにしてください

Author: 承知しました。

- Change the password of the keycloak DB (PostgreSQL) instance.
- Transfer the registered SSH key to the Bastion host using SCP, etc., place it in an appropriate directory, and set the permissions.

> The sample script "basion-setup-sample.sh" is used to install the tools that are necessary to install Vantiq on a Bastion server.

Reviewer: ```suggestion
The sample script "basion-setup-sample.sh" is used to install the tools that are necessary to install Vantiq on a Bastion host.
```

Reviewer: Bastion ServerはすべてBastion Hostとしてください。
